src/Donation/Donation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace Give\Donation;
+
+/**
+ * Class Donation
+ *
+ * @package Give\Donation
+ */
+class Donation {
```

Comment:
What is your intention for this class? Is this a model?

Reply:
@JasonTheAdams Use this is a modal. I added basic as per requirement. Let me know if you want suggest to add more properties.

Reply:
@JasonTheAdams Donation model requires discussion and research because currently, it stores in WP `post` table. For now, I remove it: https://github.com/impress-org/givewp/commit/0c4d39db3ea31828d757fd0eb64b503d85950cb2